docs/source/install.md

```diff
@@ -252,64 +275,92 @@ until [ -z "$(docker service ls --filter label=com.docker.stack.namespace=secret
 done
 if [ "$limit" -lt 0 ]; then
     echo "Not all services in the existing docker stack have been removed."
-    echo "Please wait (or try to remove them manually) and run this command again."
+    echo "Please wait (or try to remove them manually) and run this script again."
+    exit 1
 fi
+
 limit=30
 until [ -z "$(docker network ls --filter label=com.docker.stack.namespace=secrets_test -q)" ] || [ "$limit" -lt 0 ]; do
   sleep 2
   limit="$((limit-1))"
 done
 if [ "$limit" -lt 0 ]; then
     echo "Not all networks in the existing docker stack have been removed."
-    echo "Please wait (or try to remove them manually) and run this command again."
+    echo "Please wait (or try to remove them manually) and run this script again."
+    exit 1
 fi
 
+
 # Remove existing secrets
 echo "Removing existing secrets"
+
 docker secret rm FLOWMACHINE_FLOWDB_PASSWORD || true
 docker secret rm FLOWMACHINE_FLOWDB_USER || true
 docker secret rm FLOWAPI_FLOWDB_PASSWORD || true
 docker secret rm FLOWAPI_FLOWDB_USER || true
-docker secret rm POSTGRES_PASSWORD || true
+docker secret rm FLOWDB_POSTGRES_PASSWORD || true
```

Comment:
I'm definitely happy to rename `POSTGRES_PASSWORD` to `FLOWDB_POSTGRES_PASSWORD` (or `FLOWDB_ADMIN_PASSWORD` or similar), but I can't see any changes in the flowdb code that would reflect this renaming. Does it pick up the correct secret file? (Haven't tried it yet.)

Reply:
Never mind, I assume the fact that `POSTGRES_PASSWORD_FILE` is set to the correct value will suffice.

Reply:
I'm still a little confused around this though - it's a bit difficult to follow the various env vars and/or secrets being passed around via different names for the variables or secrets.

Also, it just occurred to me that the changelog mentions that `POSTGRES_PASSWORD_FILE` was renamed to `POSTGRE_PASSWORD`. So I assume something is broken somewhere?

Reply:
Ah it is - with the secrets in the Postgres image, the name of the secret is the name of a file, and you then set `POSTGRES_PASSWORD_FILE` to the path to that file. Iirc, we (I) had incorrectly interpreted this as 'you should call the secret `POSTGRES_PASSWORD_FILE`', and fixing that is what the changelog is alluding too.

Reply:
Ah, I forgot/missed that this is (i) a thing that is already present in the Postgres docker image, so it won't show up in _our_ source code, and (ii) is they add the `_FILE` suffix to the env var name via a generic function in their `docker-entrypoint.sh` script, so you can't even search for `POSTGRES_PASSWORD_FILE` in their docker setup. 😂